src/main/java/io/unlogged/command/ServerMetadata.java

```diff
@@ -11,11 +13,24 @@ public void setAgentServerPort(int agentServerPort) {
     String includePackageName;
     String agentVersion;
     String agentServerUrl;
+	private String hostname;
+	private long createdAt;
+
     public ServerMetadata(String includePackageName, String agentVersion, int agentServerPort) {
         this.includePackageName = includePackageName;
         this.agentVersion = agentVersion;
+		this.createdAt = System.currentTimeMillis() / 1000L;
         this.agentServerPort = agentServerPort;
         this.agentServerUrl = "http://localhost:" + String.valueOf(agentServerPort);
+
+		InetAddress inetAddress;
+		try {
+			inetAddress = InetAddress.getLocalHost();
+			this.hostname = inetAddress.getHostName();
+		} catch (UnknownHostException e) {
+			e.printStackTrace();
```

Comment:
send to log.txt via logger

Reply:
Done, also moved this flow a little downward since errorLogger is accessible a little later.